### PR TITLE
Added diffKeys collection method documentation

### PR DESCRIPTION
**Pull request by Ilyes512**
I was going trough the collection class and I noticed this method (that I needed at that point). It was missing from the collection's documentation page.

### CODE REVIEW

File: collections.md [hunk 1]
```diff
@@ -253,7 +254,7 @@ The `count` method returns the total number of items in the collection:
 <a name="method-diff"></a>
 #### `diff()` {#collection-method}
 
-The `diff` method compares the collection against another collection or a plain PHP `array`:
+The `diff` method compares the collection against another collection or a plain PHP `array` based on it's values:
```
Comment (vlakoff) [inline collections.md]: "its"!!!

File: collections.md [hunk 2]
```diff
@@ -263,6 +264,30 @@ The `diff` method compares the collection against another collection or a plain
 
     // [1, 3, 5]
 
+<a name="method-diffkeys"></a>
+#### `diffKeys()` {#collection-method}
+
+The `diffKeys` method compares the collection against another collection or a plain PHP `array` based on it's keys:
```
Comment (vlakoff) [inline collections.md]: God just has killed a second kitten.

File: collections.md (hunk 1 above)
Comment (Ilyes512) [inline collections.md]: Tnx, I Will fix it and make a new PR later today

Comment (vlakoff) [inline collections.md]: I don't think its worth a PR.

**Pull request merged**